### DISCUSSION TITLE
Bug report: GitHub release descriptions silently changed from optional to required

### DISCUSSION THREAD

**Discussion by tedmiston** [community/community#88789] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Hello - I use the GitHub releases feature every day.  I manage a repo where the description field is irrelevant, and it has never been required before on GitHub.

I have cut hundreds of releases over the years without a description field, including once just yesterday.  However, something seems to have been deployed that is using client-side validation to reject creating a release if a description is not provided.

Please help escalate this bug to get attention from GitHub and get this bug fixed!

<img width="1792" alt="Screen Shot 2024-01-18 at 9 41 33 PM" src="https://github.com/community/community/assets/366688/6f7b4f5e-a153-49fe-accb-9bdcce102805">

**Comment by KlutzyBubbles** [comment]
Please let this not be a permanent thing, the usefulness of publishing a release then writing the description after needs to come back.

**Comment by Estecka** [comment]
This change looks intentional to me, so I would have tagged the thread with `Product Feedback` instead of `Bug`

Ommit the release note entirely does have uses. Releases with no description automatically use the commit's message instead, which is precisely what I'm looking for.  On my personal repositories, I have a one-commit-per-release policy; the release notes are already embedded in the commit message, and I didn't need to do _anything at all_ to have this note carry over to the release page.

I'm aware the website interface has a button to automatically generate release notes, but I find its result to be lackluster. I don't see myself using it even today.

**Comment by luigifab** [comment]
Yes, please allow again to release without description.

Moreover, you can also remove the auto generated text "Version x.x.x" here:
![image](https://github.com/community/community/assets/31816829/eb6ce801-d0e4-4e09-ab31-31387237e0bb)

**Comment by Estecka** [comment]
For anyone interested, it all hinges an a simple `required=""` html tag on the textarea, so hacking around it is trivial.

**Comment by Estecka** [comment, marked as answer]
Looks like this has been fixed !

**Reply by tedmiston** [reply]
i'm marking this as the "answer" for search purposes.  i'm very glad github resolved this bug!
